fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/ExpressionMapping.java

```diff
@@ -75,7 +75,6 @@ public void setFieldMappings(List<ColumnRefOperator> fieldMappings) {
     public void put(Expr expression, ColumnRefOperator variable) {
         if (expression instanceof FieldReference) {
             fieldMappings[((FieldReference) expression).getFieldIndex()] = variable;
-            return;
```

Comment:
please desc why remove this line could fix this bug

Reply:
All the translated expr should be put into the mapping relationship expressionToColumns, here is a direct return, the corresponding relationship is omitted, resulting in fieldreference parsing error